Plugins/Bitbucket/BitbucketPlugin.cs

```diff
@@ -12,6 +13,8 @@ public class BitbucketPlugin : GitPluginBase
         public readonly StringSetting BitbucketBaseUrl = new StringSetting("Specify the base URL to Bitbucket", "https://example.bitbucket.com");
         public readonly BoolSetting BitbucketDisableSsl = new BoolSetting("Disable SSL verification", false);
 
+        private readonly TranslationString _yourRepositoryIsNotInBitbucket = new TranslationString("Your repository is not hosted in Bitbucket.");
```

Comment:
Should say BitBucket Server (previous Stash, not the same as bitbucket.org).
(This is not a new string, same in existing code.)

Reply:
Done.